packages/camera/camera_windows/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.2.6
```

Comment:
For versions <1, Dart convention is to use the build number for bugfixes, so this should be `0.2.5+1`.

Reply:
I have fixed the versioning in 4ffbedd